apps/indexers/nft-marketplace/src/archiveReq.ts

```diff
@@ -0,0 +1,20 @@
+import fetch from 'node-fetch';
+import { IBatchResponse } from './types/batch';
+
+export async function archiveReq(url: string, query: string): Promise<IBatchResponse> {
```

Comment:
We have a small util lib for gql requests: `https://github.com/subsquid/squid/blob/master/util/util-internal-gql-request/src/gql-request.ts`

It has built-in retry with backoffs support